aldryn_newsblog/cms_wizards.py

```diff
@@ -17,6 +18,22 @@
 from .models import Article
 
 
+def get_published_app_configs():
+    """
+    Returns a list of app_configs that are attached to a published page.
+    """
+    published_configs = []
+    for config in NewsBlogConfig.objects.all():
```

Comment:
Change ``NewsBlogConfig.objects.all()`` to ``NewsBlogConfig.objects.iterator()``